tests/event/test_event_storage.py

```diff
@@ -0,0 +1,40 @@
+import tempfile
+
+import pytest
+
+from hathor.event.storage.memory_storage import EventMemoryStorage
+from hathor.event.storage.rocksdb_storage import EventRocksDBStorage
+from hathor.storage.rocksdb_storage import RocksDBStorage
+from tests import unittest
+from tests.utils import HAS_ROCKSDB, generate_mocked_event
+
+
+class EventStorageBaseTest(unittest.TestCase):
+    __test__ = False
+
+    def test_save_event_and_retrieve(self):
```

Comment:
Maybe add also a `self.event_storage.get_event(-1)` just to guarantee that getting the event with a random id won't raise any errors, just return `None`.